### PR TITLE
Improved Meet the Team page

### LINKED ISSUE #92: Nicer looking Meet the Team Page
Divide this page into two sections: 'Leadership Team' and 'Members'. All the people will be listed via their github profiles. The profiles will be pulled from the list of contributors from the current slp's repo.

### LINKED ISSUE #126: Update E-board members
The E-board members on the Meet the Team page should be updated to match the E-board for the Fall 2020-Spring 2021 semesters.

### PR DESCRIPTION
**Pull request by carpenterd777**
Closes #92 
Closes #126 

I grabbed profile pictures for some of the E-board from Slack, but I was not able to find good pictures for Anjali or Abdullah this way. If any of you want to send a different picture than I found my way (especially Anjali or Abdullah), I'll swap out the old picture for the one you send me. We can also talk about a different design if we don't want our pictures on the club site.

Here are some screenshots so you don't have to pull this branch down if you don't want to:
Screen Shot 2020-05-31 at 5 19 28 PM

Screen Shot 2020-05-31 at 5 19 51 PM

The profiles have a hover style:
Screen Shot 2020-05-31 at 5 20 00 PM

### CODE REVIEW

**Comment by dyc3** [comment]
I feel like the eboard pictures are too big. What if we put them all on one row

**Comment by carpenterd777** [comment]
I can see the appeal, but the descriptions of each role might have to be nixed for the design to not look cluttered.

Screen Shot 2020-05-31 at 5 33 02 PM

**Comment by dyc3** [comment]
I don't think the descriptions are necessary. We should also try to get the names to not line wrap, or if that's not possible, center align them.

**Comment by carpenterd777** [comment]
Screen Shot 2020-05-31 at 5 51 32 PM

And on mobile:
Screen Shot 2020-05-31 at 5 52 27 PM

**Comment by dyc3** [comment]
Definitely too big on mobile. Try moving the names and roles to the right side of the photos, and making the photos smaller, so that they are wider than they are tall.

**Comment by grinchpal** [comment]
image
image
I just remembered to do this when I was thinking about directing freshmen to our website at Flock Party lol here are the screenshots for what the changes look like

File: sec.club/src/views/Team/Team.scss [hunk 1]
```diff
@@ -0,0 +1,83 @@
...
+        @media (max-width: $sm-max) {
+            height: 150px;
+            width: 150px;
+        }
+    }
+}
+
+.profile:hover {
+    img {
+        filter: brightness(50%)
```
Comment (dyc3) [inline sec.club/src/views/Team/Team.scss:56]: This looks very strange.

File: sec.club/src/views/Team/Team.scss [hunk 2]
```diff
@@ -0,0 +1,83 @@
...
+.desc {
+    margin-top: 2px;
+}
+
+.contrib-header {
+    margin-bottom: 10px;
+}
+
+li a{
+    color: #f33b31; //I wasnt sure where to find the original color for links, so I color picked it
```
Comment (dyc3) [inline sec.club/src/views/Team/Team.scss:82]: I'll fix this in a seperate PR after this gets merged in

File: sec.club/src/views/Team/Team.scss (hunk 1 above)
Comment (grinchpal) [reply sec.club/src/views/Team/Team.scss:56]: Should there be no filter on hover? Or something else on hover?

Comment (dyc3) [reply sec.club/src/views/Team/Team.scss:56]: I think it'll look better without any filter on hover

**Pull request merged**